.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval"
+    ],
+    "dependencyDashboard": true,
```

Comment:
```suggestion
```

You can drop this, if you want, because the `config:recommended` preset already enables the Dependency Dashboard.